.gitignore

```diff
@@ -112,4 +112,7 @@ dist
 .yarn/cache
 .yarn/unplugged
 .yarn/build-state.yml
-.pnp.*
\ No newline at end of file
+.pnp.*
```

Comment:
I added .idea/ to .gitignore because it was generated when I opened the project in Rider. Not sure why it touched the pnp part but it is still identical